src/contract_wrappers/proxy_wrapper.ts

```diff
@@ -0,0 +1,36 @@
+import * as _ from 'lodash';
+import {Web3Wrapper} from '../web3_wrapper';
+import {ContractWrapper} from './contract_wrapper';
+import * as ProxyArtifacts from '../artifacts/Proxy.json';
+import {ProxyContract} from '../types';
+
+/**
+ * This class includes the functionality related to interacting with Proxy contract.
+ */
+export class ProxyWrapper extends ContractWrapper {
+    private _proxyContractIfExists?: ProxyContract;
+    constructor(web3Wrapper: Web3Wrapper) {
+        super(web3Wrapper);
```

Comment:
If all the constructor is doing is calling `super`, we don't need it. Please remove.